pkg/distro/fedora/images.go

```diff
@@ -50,6 +50,10 @@ func osCustomizations(
 	osc.ExcludeBasePackages = osPackageSet.Exclude
 	osc.ExtraBaseRepos = osPackageSet.Repositories
 
+	if !t.rpmOstree {
+		osc.MarkPackages = true
+	}
```

Comment:
I think that the correct way would be to unconditionally set `osc.MarkPackages = imageConfig.MarkPackages`.

Then set to `true` in https://github.com/osbuild/images/blob/d9e892f0d7f7063a71862f6a20ca152a9d68442f/pkg/distro/fedora/distro.go#L404-L407 and overridden for edge images. There may be probably a more elegant way, but the thing is that now, the value can't be actually configured via `ImageConfig`.

In addition, it may make sense to extend the `osCustomizations()` in each distro even if not used there, to prevent any surprises in the future (since `ImageConfig` and OS pipeline implementation are used globally).